web/src/typeahead_helper.ts

```diff
@@ -199,10 +199,14 @@ export let render_stream = (stream: StreamData): string =>
         stream,
     });
 
-export const render_stream_topic = (topic_object: TopicSuggestion): string =>
+export const render_stream_topic = (
+    topic_object: TopicSuggestion,
+    is_empty_stream_topic: boolean,
+): string =>
     render_typeahead_item({
```

Comment:
Do you think we should do the following operations in this function?
```
            item.topic = util.get_final_topic_display_name(item.topic);
            const is_empty_string_topic = item.topic === "";
```

Will help to avoid code duplication if we need to reuse `render_stream_topic` in future.